tools/FileTestDefs.sh

```diff
@@ -153,6 +153,7 @@ long_lines() {
               'Doxyfile.in$' \
               'containers/Dockerfile.travis$' \
               'docs/MainSite/Main.md' \
+              'docs/CONTRIBUTING.md' \
```

Comment:
I don't think you need this.  Lines containing links are already whitelisted, but unfortunately will show up in the report if something else is wrong.  Running the checks on this file doesn't actually seem to report anything, though.